lab-khalid/lib/error-middleware.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('employee:error-middleware');
+
+module.exports = function(err, req, res, next){
+  debug('error-middleware');
+  if(err.status){
+    debug('User error');
+    res.status(err.message).send(err.message);
```

Comment:
you are incorrectly sending `err.message` back as the status and not the `err.status`. This is currently causing your error messages to fail with a 500 status because you are passing something that isn't a status but is actually a message.